spark/src/main/scala/org/apache/spark/sql/delta/stats/DataSkippingReader.scala

```diff
@@ -603,17 +611,35 @@ trait DataSkippingReaderBase
    * Returns an expression to access the given statistics for a specific column, or None if that
    * stats column does not exist.
    *
-   * @param statType One of the fields declared by object `DeltaStatistics`
-   * @param pathToColumn The components of the nested column name to get stats for.
+   * @param pathToStatType Path components of one of the fields declared by the `DeltaStatistics`
+   *                       object. For statistics of collated strings, this path contains the
+   *                       versioned collation identifier. In all other cases the path only has one
+   *                       element. The path is in reverse order.
```

Comment:
Why in reverse order? It's not clear from the names that this is the case -- this information is kind of tucked away as if it's a secret. :)

Reply:
This is to keep it consistent with pathToColumn. This way we are using foldRight on both of them, which was my main goal here